CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [0.8.5] - 2024-14-02
+
+### Added
+
+- Docker container and CI workflow for `java-spiffe-helper` (#187)
+
+### Changed
+
+- Updated Gradle to version 8.5 (#201)
+- Various enhancements in `java-spiffe-helper` (#199)
+
+### Fixed
+
+- Addressed a Fat Jar Assembly issue. (#198)
+
+### Dependencies updates
+
+- Bump `grpcVersion` from 1.54.0 to 1.61.1 (#202)
+- Bump `commons-validator:commons-validtor` from 1.7.0 to 1.8.0 (#197)
```

Comment:
```suggestion
- Bump `commons-validator:commons-validator` from 1.7. to 1.8.0 (#197)
```

- typo in package name
- previous version didn't have a patch component - just `1.7` vs. `1.7.0`